src/Security/Jwks/Fetcher/JwksFetcherInterface.php

```diff
@@ -26,5 +26,8 @@
 
 interface JwksFetcherInterface
 {
+    // TTL
```

Comment:
I would highlight that it's the TTL **in seconds**. WDYT?

Reply:
applied (full repo code base, ther TTL constants in other classes were also expressed)